### ISSUE TITLE
Reduce the length of your profile URL

### ISSUE THREAD

**Issue by baek-sang** [bluesky-social/social-app#2973]
https://bsky.app/profile/bsky.app

alternative

https://bsky.app/p/bsky.app
https://bsky.app/@bsky.app
https://bsky.app/bsky.app

**Comment by chrisalley** [comment]
`https://bsky.app/@bsky.app` would be consistent with the convention used by Mastodon and YouTube.

**Comment by importentstuff-sync** [comment]
Yeah, I think the urls are bit long. Especially when viewing a full post. /post/ should also be reduced, or removed if possible, for better UX.

**Comment by oaustegard** [comment]
Indeed: Assuming a reasonable dynamic routing handler is there any reason https://bsky.app/profile/austegard.com/post/3laomo57dy22z could not be https://bsky.app/austegard.com/3laomo57dy22z ?

See https://gist.github.com/oaustegard/0ecb5a371bd899dda8505af2bdc5d9ed for proposed code changes
